### PR TITLE
Add adoptable field

### PR DESCRIPTION
**Pull request by Pandapip1**
See the EIP-1 changes for description.

Changes needed:
- [x] EIP template change
- [x] EIP-1 description
- [ ] EIP-Bot implementation
- [ ] eipw validation

Closes https://github.com/ethereum/EIPs/issues/5245

### CODE REVIEW

**Comment by xinbenlv** [comment]
Cool, I am in favor of adding this field!

Only question is what's the default value in the template. 

@Pandapip1 any proposal?

**Comment by Pandapip1** [comment]
> Only question is what's the default value in the template.

`true`.

File: EIPS/eip-1.md [hunk 1]
```diff
@@ -139,6 +139,8 @@ Each EIP must begin with an [RFC 822](https://www.ietf.org/rfc/rfc822.txt) style
 
 `withdrawal-reason`: *A sentence explaining why the EIP was withdrawn.* (Optional field, only needed when status is `Withdrawn`)
 
+`resuscitatable`: *One of `true` or `false`.* (Optional field, but highly recommended)
```
Comment (SamWilsn) [inline EIPS/eip-1.md:142]: How about:

```suggestion
`adoptable`: *One of `true` or `false`.* (Optional field, but highly recommended)
```

File: EIPS/eip-1.md [hunk 2]
```diff
@@ -185,6 +187,10 @@ The `created` header records the date that the EIP was assigned a number. Both h
 
 EIPs may have a `requires` header, indicating the EIP numbers that this EIP depends on.
 
+#### `resuscitatable` header
+
+EIPs may have a `resuscitatable` header. If this is `true`, then any PR that changes the `status` from `Stagnant` to any other state only requires editor approval.
```
Comment (SamWilsn) [inline EIPS/eip-1.md:192]: Unless I'm missing the point of this field, this description doesn't do it justice. I assume this is to support adopting stagnant EIPs and changing their author field? I think you should mention exactly what this lets non-authors do.

Changing from `Stagnant` to `Withdrawn` also seems weird without author approval, so I've added "non-final state".

```suggestion
EIPs may have a `resuscitatable` header. If this is `true`, then any PR changing `status` from `Stagnant` to a non-final state no longer requires author approval (instead only requiring editor approval.) Marking a proposal as `resuscitatable` allows _anyone_ to edit the author list and take over the proposal should it be abandoned.
```

**Comment by xinbenlv** [comment]
> > Only question is what's the default value in the template.
> 
> 
> 
> `true`.

I am yet debating with myself whether `true` or `false` to be default value. Default is powerful and we shall use it with caution.

**Comment by SamWilsn** [comment]
> I am yet debating with myself whether `true` or `false` to be default value. Default is powerful and we shall use it with caution.

In this case, the least permissive option should be the default if the field is omitted (so `false`.) In the template, however, I would be okay with `true` being the default. `eipw` could emit a warning if the field is missing.

---

All this said, I do think this field puts editors in the awkward position of having to decide who can adopt a proposal that's been abandoned.

The simplest credibly neutral policy we can adopt is simply approving the first PR to change the status. Then it's really up to authors to stay on top of "defending" their EIP number.

**Comment by Pandapip1** [comment]
> All this said, I do think this field puts editors in the awkward position of having to decide who can adopt a proposal that's been abandoned.

First come first serve.

**Comment by MicahZoltu** [comment]
Why is the bot still mentioning me on EIPs?  This was opened 2 days ago, which is long after my name was removed.

**Comment by Pandapip1** [comment]
Not sure what happened there...

File: EIPS/eip-1.md [hunk 3]
```diff
@@ -185,6 +187,10 @@ The `created` header records the date that the EIP was assigned a number. Both h
 
 EIPs may have a `requires` header, indicating the EIP numbers that this EIP depends on.
 
+#### `adoptable` header
+
+EIPs may have a `adoptable` header. If this is `true`, then any PR changing `status` from `Stagnant` to a non-final state no longer requires author approval (instead only requiring editor approval.) Marking a proposal as `adoptable` allows _anyone_ to edit the author list and take over the proposal should it be abandoned, and is recommended.
```
Comment (xinbenlv) [inline EIPS/eip-1.md:192]: I'd love to see a more clear description on what is considered "abandoned"

Comment (Pandapip1) [reply EIPS/eip-1.md:192]: Stagnant <=> abandoned

**Comment by xinbenlv** [comment]
I like the current wording. LGTM now

File: eip-template.md [hunk 4]
```diff
@@ -9,6 +9,7 @@ type: <Standards Track, Meta, or Informational>
 category (*only required for Standards Track): <Core, Networking, Interface, or ERC>
 created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
 requires (*optional): <EIP number(s)>
+adoptable (*optional): true
```
Comment (xinbenlv) [inline eip-template.md:12]:
```suggestion
adoptable (*optional, true is recommended but not required): true
```

**Review by xinbenlv — APPROVED** [review]
Approve with optional suggestion

**Review by SamWilsn — APPROVED** [review]

**Comment by Pandapip1** [comment]
@SamWilsn the EIP-1 change requires a manual merge

**Pull request merged**